src/multitask_personalization/envs/cooking/cooking_env.py

```diff
@@ -18,26 +18,90 @@
 from multitask_personalization.structs import PublicSceneSpec
 
 
+@dataclass(frozen=True)
+class CookingPot:
+    """A pot in a cooking environment."""
+
+    radius: float = 0.25
+    depth: float = 1.0
+
+    position: tuple[float, float] | None = None
+
+
+@dataclass(frozen=True)
+class CookingIngredient:
+    """An ingredient in a cooking environment."""
+
+    name: str = "DEFAULT_INGREDIENT"
```

Comment:
this seems like we would never want to use it -- any reason to just make this a requirement argument? (so remove ` = "DEFAULT_INGREDIENT")